zipline-gradle-plugin/src/main/kotlin/app/cash/zipline/gradle/ZiplinePlugin.kt

```diff
@@ -73,10 +70,11 @@ class ZiplinePlugin : KotlinCompilerPluginSupportPlugin {
       createdTask.outputDir.set(linkOutputDir.parentFile.resolve("${linkOutputDir.name}Zipline"))
     }
 
-    project.tasks.withType(KotlinWebpack::class.java).configureEach { kotlinWebpack ->
-      if (kotlinBinary.mode.toString().equals(kotlinWebpack.mode.toString(), ignoreCase = true)) {
-        kotlinWebpack.dependsOn(compileZiplineTaskName)
-      }
+    val serveTaskName = "serve${kotlinBinary.mode.toString().lowercase().capitalized()}Zipline"
+    project.tasks.register(serveTaskName, ZiplineServeTask::class.java) { createdTask ->
```

Comment:
Can you add update docs if any of the commands for continuous development of single build have changed?

Reply:
Thanks - looking for uses of the existing webpack run points out that I've broken the samples with this PR and need to fix them.